### ISSUE TITLE
2080Ti with 11G VRAM but can't work with more than 512 * 512.

### ISSUE THREAD

**Issue by tcflying** [lstein/stable-diffusion#59]
2080Ti with 11G VRAM but can't work with more than 512 * 512. I would like to set it to 1024*1024. How To, Thanks.

**Comment by blessedcoolant** [comment]
Don't think you'd be able to churn out a 1024x1024 on that card unless someone optimizes the script a whole lot. Even on an RTX 3080, I can't do more than 512x768 => or variations in that resolution limit.

**Comment by Oceanswave** [comment]
Yeah, sorry, this is just how it is - if anything this issue should be against the upstream, as this fork has no control of that. 

On my 3090 with 24g of VRAM I'm unable to create at that resolution either

They say that the v1.5 of the weights will reduce the ram requirements further

**Comment by lstein** [comment]
There is a VRAM-optimized version of txt2img and img2img at https://github.com/basujindal/stable-diffusion. It greatly reduces memory requirements at the cost of increased execution time. You might want to give it a try. I have not yet decided whether to try to import the optimizations into this fork due to the radical nature of the differences.
